cmd/dhcp-sync/dhcp-sync.go

```diff
@@ -203,7 +203,7 @@ var Cmd = &cobra.Command{
 
 		err = watcher.Add(file)
 		if err != nil {
-			klog.Fatal(err)
+			klog.Fatalf("cannot sync DHCP server: %+v", err)
```

Comment:
is %+v required?

Reply:
Since its cleanup may be add a proper error  message above 
```
case err, ok := <-watcher.Errors:
					if !ok {
						return
					}
					klog.Error("error:", err)
				}
```

Reply:
Thanks!
1. I've made the required changes. `%+v -> %v`.
2. Added more context to the fsWatcher errror.